Stylesheets/common_css/general.css

```diff
@@ -0,0 +1,26 @@
+.container {
+    margin: 3rem auto;
+    min-width: 15rem;
+    max-width: 25rem;
+    min-height: 5rem;
+    max-height: 10rem;
```

Comment:
Just a suggestion, the clamp() function allows you to define minimum, preferred and maximum values in one line. Maybe you can shorten the code here by using that

Reply:
Thanks vanshika, didn't knew about this.. I have used it now.